build/toolchain/android/BUILD.gn

```diff
@@ -95,9 +95,17 @@ android_clang_toolchain("android_clang_x86") {
 }
 
 android_clang_toolchain("android_clang_arm") {
+  if (is_cobalt) {
+    toolchain_args = {
+      current_cpu = "arm"
+      starboard_toolchain = "//build/toolchain/android:android_clang_arm"
+      target_platform = "android-arm"
```

Comment:
this is overriden in args.gn file i thought , is it necessary to set it here?

Reply:
This is specifically for when it is used as a secondary android toolchain